backend/blueprint/api/controllers/fileUploader.py

```diff
@@ -37,16 +37,21 @@ def uploadImage():
 
 
 @router.route('/file', methods=['POST'])
-@loginRequired
 def fileUpload():
-    if 'file' not in request.files:
+    try:
+        data = request.json['csvFile']
+        csv = data['csvFile']
+    except Exception as e:
         return jsonify(
-            Response(401).generateMessage(
-                'No file is specified'))
-
-    file = request.files['file']
+            Response(400).exceptWithMessage(
+                str(e),
+                'No CSV Specified'))
+    extension = data['extension']
```

Comment:
what if extension is not present. This is outside try catch.

Reply:
Done :+1: